components/bio-formats-plugins/src/loci/plugins/About.java

```diff
@@ -44,11 +44,11 @@
 
   /** URL of the Bio-Formats web page. */
   public static final String URL_SOFTWARE =
-    "http://www.openmicroscopy.org/site/products/bio-formats";
+    "http://openmicroscopy.org/info/bio-formats";
```

Comment:
No strong veto to this change that but a mid-term implication is that this info redirect will need to be actively tested/maintained. /cc @hflynn